tequilapi/endpoints/proposals.go

```diff
@@ -101,6 +116,14 @@ func proposalToRes(p market.ServiceProposal) *proposalDTO {
 			},
 		},
 		AccessPolicies: p.AccessPolicies,
+		PaymentMethod: paymentMethodRes{
+			Type:  p.PaymentMethod.GetType(),
```

Comment:
p.PaymentMethod could be nil. For example if I build my own node to send nil payment method all consumers can start failing when fetching proposals (but not sure how handled in other places).

Reply:
It shouldn't be `nil`. It's deserialized to `market.UnsupportedPaymentMethod`.